tests/testthat/test_labels_emigration.R

```diff
@@ -3,6 +3,7 @@ context("add_labels_emigration")
 # skip tests because they take too much time
 skip_if(Sys.getenv("TEST_ONE") != "")
 testthat::skip_on_cran()
+testthat::skip_if_not_installed("arrow")
```

Comment:
Not strictly necessary for CRAN because these are already skipped there, but in case you're running locally without arrow